rotkehlchen/chain/evm/accounting/aggregator.py

```diff
@@ -123,3 +128,59 @@ def reset(self) -> None:
         """Reset the state of all initialized submodule accountants"""
         for aggregator in self.aggregators:
             aggregator.reset()
+
+
+class AccountingRuleManager:
+
+    def __init__(
+            self,
+            database: DBHandler,
+            evm_aggregators: EVMAccountingAggregators,
+            pot: 'AccountingPot',
+    ):
+        self.database = database
+        self.aggregators = evm_aggregators
+        self.pot = pot
+        self.event_settings: dict[str, BaseEventSettings] = {}
+
+    def _query_db_rules(self) -> None:
+        """"
+        """
+        query = (
+            'SELECT type, subtype, counterparty, taxable, count_entire_amount_spend, '
+            'count_cost_basis_pnl, method, accounting_treatment FROM accounting_rules'
+        )
+        with self.database.conn.read_ctx() as cursor:
+            cursor.execute(query)
+            for entry in cursor:
+                rule = BaseEventSettings.deserialize_from_db(entry[3:])
```

Comment:
perhaps we can combine this with the `get_tx_event_type_identifier` method? 

Also now that I noticed it that's quite a silly method to combine them. Why did we add two `__` between them and not one? To be able to separate them again by checking where there is a double underscore? Why just not append those strings?

Reply:
Yes, I believe is helpful for splitting them if needed. I've changed it to a single underscore and unified its usage